recipes/regreg/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "regreg" %}
+{% set version = "0.1.5" %}
+{% set sha256 = "d9d7d8f258d29ea55d57e4e8599236660837b05e04de96de36a4287cbf59b932" %}
+
+package:
+  name: regreg
+  version: {{ version }}
+
+source:
+  url: https://github.com/regreg/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Since you only want Python >=3.7, since this is not noarch, need to specify that by skipping the 3.6 builds:
```suggestion
  number: 0
  skip: true  # [py36]
```